package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "bgio-redis-pubsub",
```

Should we publish this as a scoped package to the boardgame.io org? That’s often a good a way to indicate more “official” packages.

```suggestion
  "name": "@boardgame.io/redis-pubsub",
  "private": false,
```

In that case, I would also suggest renaming this repo to just `redis-pubsub`.

---

Done

---

I am getting an error even though I've added private: false:

```
npm notice 
npm notice 📦  @boardgame.io/redis-pubsub@0.0.1
npm notice === Tarball Contents === 
npm notice 1.1kB LICENSE                    
npm notice 284B  lib/index.js               
npm notice 1.3kB lib/redis-pub-sub.js       
npm notice 2.3kB lib/redis-pub-sub.test.js  
npm notice 1.6kB package.json               
npm notice 1.1kB README.md                  
npm notice 47B   lib/index.d.ts             
npm notice 502B  lib/redis-pub-sub.d.ts     
npm notice 11B   lib/redis-pub-sub.test.d.ts
npm notice === Tarball Details === 
npm notice name:          @boardgame.io/redis-pubsub              
npm notice version:       0.0.1                                   
npm notice package size:  3.1 kB                                  
npm notice unpacked size: 8.3 kB                                  
npm notice shasum:        6f32425558fc885bd352f22d84376df4d2b2b085
npm notice integrity:     sha512-Io3AG0YIQrulV[...]SOKGH6+27HOzQ==
npm notice total files:   9                                       
npm notice 
npm ERR! code E402
npm ERR! 402 Payment Required - PUT https://registry.npmjs.org/@boardgame.io%2fredis-pubsub - You must sign up for private packages
```

---

`npm publish --access public` did it!